chainregistry.go

```diff
@@ -24,6 +24,7 @@ import (
 	"github.com/lightningnetwork/lnd/chainntnfs/bitcoindnotify"
 	"github.com/lightningnetwork/lnd/chainntnfs/btcdnotify"
 	"github.com/lightningnetwork/lnd/chainntnfs/neutrinonotify"
+	chainreg "github.com/lightningnetwork/lnd/chainregistry"
```

Comment:
why not use the full name? Or name the package chainreg if too long?

Reply:
I think it's ok if it's a commit that is meant to be deleted.